django/applications/catmaid/control/sampler.py

```diff
@@ -920,7 +922,7 @@ def add_multiple_sampler_domains(request, project_id, sampler_id):
        required: true
     """
     sampler_id = int(sampler_id)
-    domains = get_request_list(request.POST, 'domains', map_fn)
+    domains = get_request_list(request.POST, 'domains', map_fn=str)
```

Comment:
It becomes clear that this API endpoint isn't currently used by the front-end. :-) Anyway, the `map_fn` should not be `str` (which it is anyway), but we expect a JSON encoded object for each element. Therefore it should read `map_fn=lambda x: json.loads(x)`.

Reply:
Couldn't we just use `map_fn=json.loads`, in that case?

Reply:
True, that should work as well and looks nicer.

Reply:
FYI, I added this because (I believe, and mypy believes that) this syntax actually tries to do a pass-by-position argument pass rather than a keyword pass with no value. Will revise as per clbarnes.